src/client/common/application/customEditorService.ts

```diff
@@ -44,35 +44,43 @@ export class CustomEditorService implements ICustomEditorService {
         }
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    private async enableCustomEditors() {
-        // This code is temporary.
-        const settings = this.workspace.getConfiguration('workbench', undefined);
-        const editorAssociations = settings.get('editorAssociations') as {
-            viewType: string;
-            filenamePattern: string;
-        }[];
+    public async activate() {
+        let updateType: 'added' | 'removed' | undefined;
+        try {
+            // This code is temporary.
```

Comment:
Apparently not!

Reply:
Lol, i guess temporary for the long term  😆